test/cli.js

```diff
@@ -680,6 +680,49 @@ test('legacy snapshot files are reported to the console', t => {
 	});
 });
 
+test('snapshots infer their location from sourcemaps', t => {
+	t.plan(8);
+	const dirname = path.join('fixture/snapshots/test-sourcemaps');
+	const testLocationTypes = [
+		'src',
+		'src/test/snapshots',
+		'src/feature/__tests__/__snapshots__'
+	];
+	const removeExists = relFilePath => {
```

Comment:
I'm not too concerned about how DRY this is. The other test looks good, just need to take that same approach here.